core/modules/layout/layout.theme.inc

```diff
@@ -250,9 +250,20 @@ function template_preprocess_block(&$variables) {
   $style = $block->style;
 
   $variables['classes'] = array();
+  $variables['attributes'] = array();
   $variables['title_prefix'] = '';
   $variables['title_suffix'] = '';
 
+  // Add the hero background image inline. This could also be added in a
+  // specific template_preprocess_block__layout__hero() function, but we add
+  // it here just to cut down on unnecessary separation.
+  if (get_class($block) == 'BlockHero') {
+    $variables['classes'][] = 'hero-block';
+    if (isset($block->settings['image_path'])) {
```

Comment:
Would be nice to have a utility class if there isn't an image.

`hero-block-no-bg-image` or something?